src/minioInstallationCheck.ts

```
@@ -0,0 +1,79 @@
+ * This function first checks if MinIO is already installed by calling `isMinioInstalled()`.
+ * - If MinIO is found to be installed, it logs a message and resolves with no value.
+ * - If MinIO is not found, it initiates the installation process using `installMinio()`.
+ *   - If the installation succeeds, it logs a success message and resolves with the path to the installed MinIO binary.
+ *   - If the installation fails, it logs an error message and rejects the promise with the error.
+ *
+ * @returns A promise that resolves with:
+ *   - The path to the MinIO binary if it was installed.
+ *   - No value if MinIO was already installed.
+ * @throws Error If an error occurs during the check or installation process.
+ */
+export const checkMinio = async (): Promise<string | void> => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)